relate/templates/sign-in-choice.html

```diff
@@ -5,6 +5,15 @@
   <h1>{% blocktrans %}Sign in to {{ relate_site_name }}{% endblocktrans %}</h1>
 
   <ul class="list-sign-in-methods">
+    {% if relate_sign_in_by_cas_enabled %}
+      <li>
+        <a
+          class="btn btn-primary"
+          href="{% url 'cas_ng_login' %}{{next_uri}}"
+          role="button"><i class="fa fa-institution"></i>
+          {% trans "Sign in using your institution's login" %} &raquo;</a>
```

Comment:
In the event that both SAML and CAS are enabled, it might be useful to add a "(CAS)" here and a "(SAML)" to the SAML button.

Reply:
Outside the translation or inside it, though? Wouldn't want to break the translations that are already there...